modules/apps/batch-planner/batch-planner-web/src/main/resources/META-INF/resources/js/SaveTemplate.js

```diff
@@ -25,10 +26,16 @@ function SaveTemplate({
 	portletNamespace,
 }) {
 	const [disable, setDisable] = useState(true);
+	const isMounted = useIsMounted();
 	const [visible, setVisible] = useState(false);
 	const {observer, onClose} = useModal({
-		onClose: () => setVisible(false),
+		onClose: () => {
+			if (isMounted()) {
```

Comment:
I'm not sure it is necessary here. 
If you wrote it due to some warnings in a test env, keep it this way otherwise remove it, please :)